File: user-service/src/main/java/rs/edu/raf/model/user/Company.java
package rs.edu.raf.model.user;

import jakarta.persistence.Entity;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class Company extends BankAccountHolder {
    private String name;
    private String accountNumbers;
    private String faxNumber;
    private Integer TIN; // tax identification number
    private Integer registrationNumber;
    private Integer businessActivityCode;
    private Integer registryNumber;
}

File: user-service-kotlin/src/main/kotlin/org/example/userservicekotlin/model/user/Company.kt
package org.example.userservicekotlin.model.user

import jakarta.persistence.Entity

@Entity
class Company(
    id: Long?,
    phoneNumber: String,
    address: String,
    val name: String,
    var accountNumbers: String?,
    val faxNumber: String,
    val TIN: Int, // tax identification number
    val registrationNumber: Int,
    val businessActivityCode: Int,
    val registryNumber: Int
): BankAccountHolder(id, phoneNumber, address)
